grass-gis-addons/m.analyse.trees/v.trees.param/v.trees.param.py

```diff
@@ -289,6 +289,7 @@ def main():
     if nprocs > 1:
         for subset_name, mapset_name in zip(subset_names, mapset_names):
             treecrown_subset_mapset.append(f"{subset_name}@{mapset_name}")
+        grass.message(len(mapset_names))
```

Comment:
Was it only for debugging? Otherwise extend message maybe a little bit.

Reply:
only debugging

Reply:
yes, I removed it.